e2e/fx.spec.ts

```diff
@@ -50,13 +54,26 @@ test.describe("Fx App", () => {
 
     await poppedOutBlotterPage.waitForSelector("text=Trades")
 
-    await poppedOutLiveRatesPage.getByTestId("openfin-chrome__close").click()
+    await poppedOutLiveRatesPage
+      .locator("[data-qa='openfin-chrome__close']")
+      .click()
+
+    await expect(poppedOutLiveRatesPage.isClosed()).toBeTruthy()
+    await expect(poppedOutBlotterPage.isClosed()).toBeFalsy()
+    await expect(mainWindow.locator("text=Live Rates")).toBeVisible()
+    
+    await poppedOutBlotterPage
+      .locator("[data-qa='openfin-chrome__close']")
+      .click()
+    
+    await expect(poppedOutBlotterPage.isClosed()).toBeTruthy()
 
-    expect(await mainWindow.waitForSelector("text=Live Rates")).toBeTruthy()
+    await expect(mainWindow.locator("text=Trades")).toBeVisible()
+    await expect(mainWindow.locator("text=Analytics")).toBeVisible()
 
-    await poppedOutBlotterPage.getByTestId("openfin-chrome__close").click()
+    await mainWindow.getByTitle("toggle layout lock").hover()
+    await mainWindow.getByTitle("toggle layout lock").click()
 
-    expect(await mainWindow.waitForSelector("text=Trades")).toBeTruthy()
-    expect(await mainWindow.waitForSelector("text=Analytics")).toBeTruthy()
+    await expect(popOutButtons.nth(0)).not.toBeVisible()
```

Comment:
I added locking back the toggle layout as extra validation and also to reset the application to the initial state so whenever we want to run the test again it is locked as expected. Better approach would be to check for toggle state at the start of the script and act upon it

Reply:
actually, I meant that we should be running a new locator function in Playwright, surely, rather than using the old `popOutButtons` reference ... or have I missed something obvious?